docs/framework.md

```diff
@@ -42,10 +42,13 @@ if ($pharPath) {
     }
 }
 
-$commandClasses = [ \MyProject\Commands\RoboCommands::class ];
-$runner = new \Robo\Runner($commandClasses);
-$output = new \Symfony\Component\Console\Output\ConsoleOutput();
-$statusCode = $runner->execute($_SERVER['argv'], $output, 'MyAppName', '0.0.0-alpha0');
+$commandClasses = [ \MyProject\Commands\RoboFile::class ];
+$statusCode = Robo::run(
+    $_SERVER['argv'], 
+    $commandClasses, 
+    'MyAppName', 
+    '0.0.0-alpha0'
+);
```

Comment:
Yeah, that bothered me too, but adding another parameter seemed like it was making the parameter list too long. I agree with you, though; I'll put `$output = null` on the very end; then it won't bother anyone using the typical use case, and will also be testable.

I'll also fix the missing `function` typo. ;)